test/e2e/framework/test_context.go

```diff
@@ -174,6 +178,8 @@ func RegisterCommonFlags() {
 	flag.StringVar(&TestContext.OutputPrintType, "output-print-type", "json", "Format in which summaries should be printed: 'hr' for human readable, 'json' for JSON ones.")
 	flag.BoolVar(&TestContext.DumpLogsOnFailure, "dump-logs-on-failure", true, "If set to true test will dump data about the namespace in which test was running.")
 	flag.BoolVar(&TestContext.DisableLogDump, "disable-log-dump", false, "If set to true, logs from master and nodes won't be gathered after test run.")
+	flag.BoolVar(&TestContext.EnableLogexporter, "enable-logexporter", false, "If true, use logexporter tool to upload logs from nodes to GCS directly (saves time and is recommended).")
```

Comment:
Seems like we can combine this into a single flag, which specifies where we export the logs, rather than true/false